### PR TITLE
New package: devil-1.7.8

### LINKED ISSUE #2028: Package Request: DevIL
http://openil.sourceforge.net/
https://github.com/DentonW/DevIL

### PR DESCRIPTION
**Pull request by pullmoll**
Closes #2028

### CODE REVIEW

File: srcpkgs/devil/template [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		cp -aR docs/html docs/images ${PKGDESTDIR}/usr/share/doc/devil
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+	}
+}
+devil-examples_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - examples"
```
Comment (xtraeme) [inline srcpkgs/devil/template]: noarch=yes

Comment (pullmoll) [inline srcpkgs/devil/template]: No, because there are binaries installed in `/usr/share/devil/examples`.

Comment (xtraeme) [inline srcpkgs/devil/template]: Oh you are right, sorry.

**Comment by xtraeme** [comment]
:+1:

**Comment by pullmoll** [comment]
Ooops. I thought it was merged.

**Pull request merged**